web_src/less/_base.less

```
@@ -166,6 +167,8 @@
   --color-small-accent: var(--color-primary-light-6);
   --color-accent: var(--color-primary-light-4);
   --color-active-line: #fffbdd;
+
+  accent-color: var(--color-primary-light-1);
```

Shouldn't this be
```suggestion
  accent-color: var(--color-accent);
```
?
(By the way, this should also fix the problem you mentioned with the "too dark/ too light")

---

Hmm, I'm not sure `--color-accent` is 3 steps off the color I had in mind (a saturated hue).

---

Maybe we should make `--accent-color` be `--color-primary-light-1`. Thought I need to test first how current `--color-primary-light-4`, looks. I'm almost sure it will be to bright/dark.

---

Done that.